src/Constants.scss

```diff
@@ -0,0 +1,18 @@
+﻿/* Site colours */
+$blue: rgba(36, 30, 78, 1);
+$red: rgba(150, 2, 0, 1);
+$rust: rgba(206, 108, 71, 1);
+$orange: rgba(244, 162, 97, 1);
+$green: rgba(234, 218, 162, 1);
```

Comment:
This is nice - allows us to update shades really easily.

One way to make this even more flexible is to name the constants after their use rather than after their value - similar to what you did with the headers below.

Eg, instead of `$blue`, `$red` etc you could have
```sass
$fontColor: rgba(36, 30, 78, 1);
$backgroundColor: rgba(36, 30, 78, 1);
$accentColor: rgba(36, 30, 78, 1);
```

etc. This way we can totally change the whole colour scheme for the app just by updating this one file.

Reply:
(similarly with the fonts)